sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -48,21 +48,44 @@ multiple times in the input cell. The plots will display in order.
 JupyterLab
 ++++++++++
 
-To embed Bokeh plots in JupyterLab, you'll need the following two JupyterLab extensions:
+To use JupyterLab with Bokeh, you should at least use version 3.0 of JupyterLab.
+Enabling Bokeh visualizations in JupyterLab also requires the
+`jupyter_bokeh <https://github.com/bokeh/jupyter_bokeh>`_ extension to be
+installed.
 
-1. First install *jupyterlab-manager* with this command:
+After installing JupyterLab, you can use either ``pip`` or ``conda`` to install
+jupyter_bokeh:
 
-   .. code::
+.. panels::
 
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    Installing with ``conda``
+    ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It's a tiny detail but almost all of the docs already follow this convention:
```
Chapter
======

Section
-------

Sub-Section
~~~~~~~~~~~

Sub-sub Section
'''''''''''''''
```

Reply:
This is actually one of the few documents that use all four levels. In this case: 'Using with Jupyter' > 'Working in notebooks' > 'Standalone output' > 'JupyterLab'. The sub-sub sections ('Classic notebooks', 'JupyterLab', and 'JupyterHub') did indeed not follow the convention. I have fixed that. 
'Installing with ``conda``' is technically a sub-sub-sub section, but this will be displayed as part of the panels boxes:
![grafik](https://user-images.githubusercontent.com/39711796/112707451-d074f180-8e68-11eb-8455-cdd94cef78d6.png)
Those headlines are not really section headlines and are not included in the sidebar navigation. I'd leave them as they are.